MbDotNet.Acceptance.Tests/packages.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Castle.Core" version="3.3.3" targetFramework="net45" />
+  <package id="FluentAssertions" version="4.18.0" targetFramework="net45" />
+  <package id="Moq" version="4.5.28" targetFramework="net45" />
```

Comment:
Is Moq a dependency of FluentAssertions? If not, remove this since it seems like it is not used.